atqa-framework/tests/e2e/steps/InitSteps.ts

```diff
@@ -0,0 +1,495 @@
+import path from 'path';
+import expect from 'expect';
+
+import { BasePage, IterableConstant } from '../../pages/BasePage';
+import { ExtensionInitPage } from '../../pages/ExtensionInitPage';
+import { AccountPage } from '../../pages/AccountPage';
+import { ClockUnit } from '../../../utils/clockUnit';
+import { installAddOnHelper } from '../../../utils/installGeckoAddon';
+import { copyDir } from '../../../utils/copyDir';
+import {
+  getChromeExtensionVersion,
+  getOperaExtensionVersion,
+  getEdgeExtensionVersion,
+} from '../../../utils/grabExtensionVersion';
+import { ResourcesProvider } from '../../../testData/res/ResourcesProvider';
+import { DEFAULT_PASSWORD } from '../../../testData/res/constants';
+import extensionVersion from '../../../../package.json';
+import { DataGenerator } from '../../../utils/DataGenerator';
+import { UserProvider } from '../../../testData/UserProvider';
+
+const extensionInitPage = new ExtensionInitPage();
+const basePage = new BasePage();
+const accountPage = new AccountPage();
+const clockUnit = new ClockUnit();
+const resourcesProvider = new ResourcesProvider();
+const dataGenerator = new DataGenerator();
+const userProvider = new UserProvider();
+
+const { I } = inject();
+
+Given('I prepare Opera extension', async () => {
+  const extId = await basePage.getItemFromLocalStorage('operaId');
+  const popup = basePage.BROWSER_URLS.POPUP;
+  const extensionWelcomePage = extId.concat(popup);
+  await I.amOnPage(extensionWelcomePage);
+  I.waitForElement(
+    accountPage.SELECTORS.GET_STARTED_BUTTON,
+    clockUnit.SECONDS * 30
+  );
+  I.switchToNextTab(2);
+});
+
+Given('I prepare Firefox extension', async () => {
+  await installAddOnHelper(
+    path.join(__dirname, '..', '..', 'dist', 'keeper-wallet-2.9.0-firefox.xpi')
+  );
+  const extId = await basePage.getItemFromLocalStorage('firefoxId');
+  const popup = basePage.BROWSER_URLS.POPUP;
+  const extensionWelcomePage = extId.concat(popup);
+  await I.amOnPage(extensionWelcomePage);
+  I.waitForElement(
+    accountPage.SELECTORS.GET_STARTED_BUTTON,
+    clockUnit.SECONDS * 30
+  );
+  I.closeCurrentTab();
+});
+
+Given(
+  /^I on the '(POPUP|ACCOUNTS)' page for id '([^']+)'$/,
+  async (pageType: string, browserId: string) => {
+    const extId = await basePage.getItemFromLocalStorage(browserId);
+    console.info(extId);
+    const iterablePageUrls: IterableConstant = basePage.BROWSER_URLS;
+    const targetPage = extId.concat(iterablePageUrls[pageType]);
+    I.amOnPage(targetPage);
+  }
+);
+
+Given(
+  /^I on the '(CHROMIUM_EXTENSIONS|FIREFOX_EXTENSIONS|CHROME_SYSTEM)' page$/,
+  (pageType: string) => {
+    const iterablePageUrls: IterableConstant = basePage.BROWSER_URLS;
+    I.amOnPage(iterablePageUrls[pageType]);
+  }
+);
+
+When(/^I restart (?:Opera|Chrome) extension$/, () => {
+  I.waitForElement(
+    extensionInitPage.SELECTORS.CHROMIUM.SERVICE_WORKER_RESTART_BUTTON,
+    clockUnit.SECONDS * 30
+  );
+  I.forceClick(
```

Comment:
How is `forceClick` different from just `click`? Can you give a link to docs may be?